src/ng/location.js

```diff
@@ -938,8 +938,6 @@ function $LocationProvider() {
           // update location manually
           if ($location.absUrl() !== $browser.url()) {
             $rootScope.$apply();
-            // hack to work around FF6 bug 684208 when scenario runner clicks on links
-            $window.angular['ff-684208-preventDefault'] = true;
```

Comment:
This is still used in `browserTrigger`: https://github.com/angular/angular.js/pull/15931/files#diff-6fcc18d824a1325cf54ef306cd2fb7a5R124